mobile/android/base/sync_strings.dtd

```diff
@@ -22,7 +22,7 @@
 <!ENTITY bookmarks.folder.places.label "">
 <!ENTITY bookmarks.folder.tags.label "Etykiety">
 <!ENTITY bookmarks.folder.toolbar.label "Pasek zakładek">
-<!ENTITY bookmarks.folder.unfiled.label "Zakładki nieposortowane">
+<!ENTITY bookmarks.folder.other.label "Inne zakładki">
```

Comment:
Browser will use "Pozostałe zakładki" and I personally consider [notes to self](https://github.com/aviarypl/mozilla-l10n-central-pl/pull/82/commits/a5f5e3077cd9ca28503bf6e3644fb44987ab226d) harmful.